ccv-evm/impl.go

```diff
@@ -1589,3 +1620,61 @@ func GetContractAddrForSelector(addresses []string, selector uint64, contractTyp
 	}
 	return contractAddr, nil
 }
+
+// fundLockReleaseTokenPool funds a lock/release token pool by granting mint role and minting tokens.
+func (m *CCIP17EVM) fundLockReleaseTokenPool(
```

Comment:
This can mostly be done through DeployBurnMintTokenAndPool. Mint a bunch of tokens to the deployer, and then the only thing required outside the changeset is for the deployer to send funds to the pool.